android/src/main/java/com/dylanvann/fastimage/FastImageViewWithUrl.java

```diff
@@ -141,16 +146,23 @@ public void onAfterUpdate(
             }
         }
 
+        Transformation<Bitmap> transformation = new BitmapTransformation() {
+            @Override
+            protected Bitmap transform(@NonNull BitmapPool pool, @NonNull Bitmap toTransform, int outWidth, int outHeight) {
+                return toTransform;
+            }
+
+            @Override
+            public void updateDiskCacheKey(@NonNull MessageDigest messageDigest) {
+            }
+        };
+
         if (requestManager != null) {
             RequestBuilder<Drawable> builder =
                     requestManager
-                            // This will make this work for remote and local images. e.g.
-                            //    - file:///
-                            //    - content://
-                            //    - res:/
-                            //    - android.resource://
-                            //    - data:image/png;base64
```

Comment:
Please revert these lines,.

Reply:
this is not needed in future , no point in reverting